{{cookiecutter.slug}}/bootstrap.py

```diff
@@ -184,6 +183,7 @@ def modify_angular_json():
         data = json.load(file)
     try:
         project = '{{cookiecutter.slug}}'.replace('_', '-')
+        data['projects'][project]['architect']['build']['options']['deployUrl'] = '/static/'
```

Comment:
I was actually going to point you to a Stack Overflow answer that mentioned this solution. But you already found it yourself. 👍